src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  color: $color-text-card;
+
+  &:hover {
+    transform: scale(1.2);
+    transition-duration: 300ms;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+    margin-left: 3px;
+    transition: 3s;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding-top: 40px;
+    height: 36px;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+    cursor: pointer;
+    transition: 3s;
```

Comment:
The styling here is right, but a 3s transition for a hover, are you sure? Maybe you should check it in the [Checklist](https://github.com/mate-academy/layout_catalog#:~:text=Make%20all%20the%20changes%20smooth%20on%20hover%20(during%20300ms)%3A)